fill-archivedwebsites-json.py

```diff
@@ -11,11 +11,9 @@ def listdirs(folder):
 
 directories = listdirs('/var/www/vhosts/archiveweb.epfl.ch/htdocs/')
 
-if 'common' in directories:
-    directories.remove('common')
+directories_to_remove = [ 'common', 'templates', 'fonts', 'cdn-cgi' ]
```

Comment:
Good, just pep8 convention to fix:

./fill-archivedwebsites-json.py:14:26: E201 whitespace after '['
./fill-archivedwebsites-json.py:14:68: E202 whitespace before ']'